### ISSUE TITLE
Show ref in changeset summaries when no name is available

### ISSUE THREAD

**Issue by bigbug21** [openstreetmap/openstreetmap-website#1140]
With hundreds of thousands of railway-related nodes in the OSM database -- such as milestones, signals and switches -- it would be great to actually see those objects when browsing through changeset summaries. For example, it would be good to see "railway signal XYZ" instead of a random node id.

It would help a lot if a 'ref' tag would be displayed when no 'name' tag is available.

**Comment by simon04** [comment]
https://github.com/openstreetmap/iD uses `iD.util.displayName` to name the objects in the summary:

![2016-03-08-22-01-23-001](https://cloud.githubusercontent.com/assets/782446/13616260/63b4c5ea-e579-11e5-8736-68c37c469f5a.png)

```js
iD.util.displayName = function(entity) {
    var localeName = 'name:' + iD.detect().locale.toLowerCase().split('-')[0];
    return entity.tags[localeName] || entity.tags.name || entity.tags.ref;
};
```

**Comment by mmd-osm** [comment]
This should be fixed as part of  #1175
